pcsx2/CMakeLists.txt

```diff
@@ -326,6 +326,118 @@ set(pcsx2DEV9Headers
 	${pcsx2DEV9UIHeaders}
 	)
 
+# USB sources 
+set(pcsx2USBSources
+	USB/USB.cpp
+	USB/deviceproxy.cpp
+	USB/configuration.cpp
+	USB/osdebugout.cpp
+	USB/device_init.cpp
+	USB/qemu-usb/glib.cpp
+	USB/qemu-usb/vl.cpp
+	USB/qemu-usb/iov.cpp
+	USB/qemu-usb/desc.cpp
+	USB/qemu-usb/core.cpp
+	USB/qemu-usb/bus.cpp
+	USB/qemu-usb/usb-ohci.cpp
+	USB/qemu-usb/hid.cpp
+	USB/qemu-usb/input-keymap-qcode-to-qnum.cpp
+	USB/usb-msd/usb-msd.cpp
+	USB/usb-pad/usb-pad.cpp
+	USB/usb-pad/usb-pad-ff.cpp
+	USB/usb-pad/lg/lg_ff.cpp
+	USB/usb-mic/usb-mic-singstar.cpp
+	USB/usb-mic/usb-mic-logitech.cpp
+	USB/usb-mic/usb-headset.cpp
+	USB/usb-eyetoy/jpgd/jpgd.cpp
+	USB/usb-eyetoy/jo_mpeg.cpp
+	USB/usb-eyetoy/usb-eyetoy-webcam.cpp
+	USB/usb-hid/usb-hid.cpp
+	USB/shared/shared.cpp
+	USB/shared/inifile.cpp
+	USB/shared/ringbuffer.cpp
+	USB/icon_buzz_24.cpp
+	USB/usb-msd/usb-msd-gtk.cpp
+	USB/usb-pad/joydev/joydev.cpp
+	USB/usb-pad/joydev/joydev-gtk.cpp
+	USB/usb-pad/evdev/shared-gtk.cpp
+	USB/usb-pad/evdev/evdev-ff.cpp
+	USB/usb-pad/evdev/evdev.cpp
+	USB/usb-pad/evdev/evdev-gtk.cpp
+	USB/usb-hid/evdev/evdev-gtk.cpp
+	USB/usb-eyetoy/cam-linux.cpp
+	USB/linux/config.cpp
+	USB/linux/config-gtk.cpp
+	USB/linux/util.cpp
+	USB/qemu-usb/input-keymap-linux-to-qcode.cpp
+	USB/usb-mic/audiodev-pulse.cpp
+    USB/usb-pad/api_init_linux.cpp
+    USB/usb-eyetoy/api_init_linux.cpp
+    USB/usb-hid/api_init_linux.cpp
+    USB/usb-mic/api_init_linux.cpp
```

Comment:
Formatting, some at bottom as well.

Reply:
Just noting but the merge is far from done, I haven't even made a clang-format run yet, so I expected to do this later down the road